src/main/java/org/myrobotlab/service/InMoov2.java

```diff
@@ -1842,6 +1842,31 @@ public InMoov2Torso startTorso(String port) {
     return torso;
   }
 
+  public Pir startPir() {
+    return startPir(null);

```

Comment:
we need to choose a default "port" and "pirPin" value here so that it calls the method startPir(port, pirPin) as defined below.

Reply:
ok thanks i will add that

Reply:
and if I remove?
  public Pir startPir() {
    return startPir(null);
  }

Reply:
you can remove the whole startPir() method if you like.. sure..